### PR TITLE
[TD-361] First draft of Helm Chart changes

### PR DESCRIPTION
**Pull request by marksou**
I've done the following:

- Created helm chart pages within the OSS and Self-Managed section.
- Added content as in the Helm Repo
- Added appropriate sections within the existing TIB, MDCB, and Tyk Cloud Hybrid Gateway pages
- Moved the Istio page under Kubernetes

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-master-data-centre.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Or
 ```{.copyWrapper}
 sudo yum install tyk-sink
 ```
+## Installing in a Kubernetes Cluster with our Helm Chart
+
+If you are deploying the Master Data Centre in an MDCB deployment then you can set the `mdcb.enabled` option in your `vuales.yaml` to true to add the MDCB component to your cluster.
```
Comment (craigtyk) [inline tyk-docs/content/tyk-stack/tyk-multi-data-centre/setup-master-data-centre.md]: correct to `values.yaml`

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md [hunk 2]
```diff
@@ -13,29 +13,24 @@ aliases:
...
 
-There are two main ways to install Tyk on Kubernetes: Via our Helm chart, or manually.
+We have two methods of installing a Tyk Self-Managed installation in a Kubernetes cluster.
 
----
 
 ## Tyk Helm Chart
 
-This is the preferred (and easiest) way to install Tyk Pro on Kubernetes. It will install Tyk as an ingress to your K8s cluster, where you can then add new APIs to manage via our Tyk Kubernetes Operator,  or as with a normal Tyk Pro Installation managed with a Dashboard Control Plane.
+This is the preferred (and easiest) way to install Tyk Pro on Kubernetes. It will install Tyk in your Kubernetess cluster, where you can then add APIs to manage via either our Tyk Kubernetes Operator,  or as with a normal Tyk Pro Installation managed with a Tyk Dashboard.
```
Comment (craigtyk) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md]: Remove extra 's' at the end of the 2nd Kubernetes

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+tags: [""]
+description: ""
+menu:
+  main:
+    parent: "On Kubernetes"
+weight: 1
+url: "/tyk-self-managed/tyk-helm-chart"
+---
+
+## Introduction
```
Comment (craigtyk) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md]: Do we need further Introduction text beyond what's explained on parent page? @letzya @sedkis 
If not, we can remove Introduction header

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+## Tyk Licensing
+
+If you are evaluating Tyk on Kubernetes, [contact us](https://tyk.io/about/contact/) to obtain an temporary licence.
```
Comment (craigtyk) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md]: Correct an -> a

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+Tyk Pro Licenses allow for different numbers of Gateway nodes to connect to a single Dashboard instance. Ensure that your Gateway pods will not scale beyond your license number by setting the Gateway resource kind to Deployment and setting the replica count to your license node limit. For example, use the following options for a single node license: `--set gateway.kind=Deployment --set gateway.replicaCount=1` in your `values.yaml` file.
+
+{{< note success >}}
+**Note**  
+
+There may be intermittent issues on the new pods during the rolling update process, when the total number of online gateway pods is more than the license limit with lower amounts of Licensed nodes.
+
+{{< /note >}}
+
+## Prerequisites
```
Comment (craigtyk) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md]: Pleased to see clear prerequisites on each page. Thanks

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+Please note the `--wait` argument is important to successfully bootstrap your Tyk Dashboard.
+
+Follow the instructions in the Notes that follow the installation to find your Tyk login credentials.
+
+## Sharding APIs
+
+Sharding is the ability for you to decide which of your APIs are loaded on which of your Tyk Gateways. This option is turned off by default, however, you can turn it on by updating the `gateway.sharding.enabled option`. Once you do that you will need to also populate the `gateway.sharding.tags` value with the tags that you want that particular Gateway to load. (ex. tags: "external,ingress".) You can then add those tags to your APIs in the API Designer, under the **Advanced Options** tab, and the Segment Tags (Node Segmentation) section in your Tyk Dashboard. See 
+
+## Other Tyk Components
```
Comment (craigtyk) [inline tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-helm-chart.md]: Agree with moving the additional components details under their respective sections under Tyk Stack to minimise duplication.

File: tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+tags: [""]
+description: ""
+menu:
+  main:
+    parent: "Kubernetes"
+weight: 1
+url: "/tyk-oss/ce-helm-chart/"
+---
+
+## Introduction
```
Comment (craigtyk) [inline tyk-docs/content/tyk-stack/tyk-gateway/install/ce-helm-chart.md:13]: Additional intro needed beyond parent page summary? If not, header can be removed

**Review by craigtyk — CHANGES_REQUESTED** [review]
Updated structure is good and added pre-reqs help clarify what's needed. Some minor corrections requested but OK from my side to publish when complete.

File: tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/tyk-on-prem/installation/with-kubernetes/with-kubernetes.md]: Done

File: tyk-docs/content/tyk-on-prem/installation/kubernetes/kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  - /tyk-on-premises/kubernetes
+---
+
+## Installing Tyk
+
+The main ways to install *Tyk Self-Managed* in a Kubernetes cluster are via Helm charts or via Kubernetes manifest files.
+
+### Tyk Helm Charts
+
+We will explain in detail the installation using [Helm Charts]({{< ref "/content/tyk-on-prem/installation/kubernetes/helm-chart.md" >}})
```
Comment (letzya) [inline tyk-docs/content/tyk-on-prem/installation/kubernetes/kubernetes.md]: This link doesnt work

File: tyk-docs/content/tyk-on-prem/installation/kubernetes/helm-chart.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+menu:
+  main:
+    parent: "Kubernetes "
+weight: 1
+url: "/tyk-self-managed/tyk-helm-chart"
+---
+
+## Introduction
+
+This is the preferred (and easiest) way to install Tyk Self-Managed Pro on Kubernetes.
```
Comment (letzya) [inline tyk-docs/content/tyk-on-prem/installation/kubernetes/helm-chart.md]: Self-Managed Pro?

File: tyk-docs/content/tyk-on-prem/installation/kubernetes/helm-chart.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+{{< warning  success >}}
+**Warning**
+
+To get started quickly, you can use `redis.yaml` and `mongo.yaml` manifests to install Redis and MongoDB inside your Kubernetes cluster. These provided manifests must **never** be used in production and are for a quick start evaluation purpose only. Use external DBs or Official Helm charts for MongoDB and Redis in any other case. We provide these manifests so you can quickly have Tyk up and running, however they are not meant for long term storage of data.
+{{< /warning >}}
+
+```{copy.Wrapper}
+kubectl create namespace tyk
+kubectl apply -f deploy/dependencies/mongo.yaml -n tyk
+kubectl apply -f deploy/dependencies/redis.yaml -n tyk
```
Comment (letzya) [inline tyk-docs/content/tyk-on-prem/installation/kubernetes/helm-chart.md]: this is the old doc @marksou

**Pull request merged**